pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -927,6 +928,32 @@ func (service *SecurityPolicyService) buildRulePeerGroup(obj *v1alpha1.SecurityP
 	return &rulePeerGroup, rulePeerGroupPath, nil, err
 }
 
+// dedupBlocks removes duplicated IPBlock CIDRs from the rule peers.
+func (service *SecurityPolicyService) dedupBlocks(rulePeers []v1alpha1.SecurityPolicyPeer) []v1alpha1.SecurityPolicyPeer {
```

Comment:
Where do we check one ipblock cidr is subnet of another ipblock cidr?

Reply:
The NSX operator directly passed the cidrs to the NSX, the NSX can accept this kind of combo.
<img width="437" height="186" alt="image" src="https://github.com/user-attachments/assets/8d991ceb-f268-4723-a031-6237d53cca41" />